x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_table/intance_details.tsx

```diff
@@ -52,7 +73,19 @@ const serviceDetailsKeys = [
   SERVICE_RUNTIME_NAME,
   SERVICE_RUNTIME_VERSION,
 ];
-const containerDetailsKeys = [CONTAINER_ID, HOST_NAME, POD_NAME];
+const containerDetailsKeys = [
+  CONTAINER_ID,
+  HOST_NAME,
+  KUBERNETES_POD_UID,
+  KUBERNETES_POD_NAME,
+];
+const metricsKubernetesDetailsKeys = [
```

Comment:
The idea is to differentiate the fields coming from metrics indices in order to disable the filter by value option in the accordion. 

Since we don't store these fields in apm the filter by value won't work for the `metricsKubernetesDetailsKeys`